File: java-app/src/main/java/com/mercadolibre/java_app/commons/Prices.java
package com.mercadolibre.java_app.commons;

import java.text.NumberFormat;
import java.util.Locale;

public class Prices {
    public static String priceFormat(Double price) {
        NumberFormat decimalFormat = NumberFormat.getNumberInstance(Locale.GERMAN);
        return "$ " + decimalFormat.format(price);
    }
}

File: kt-app/src/main/java/com/mercadolibre/melikotlinmeetup/commons/Prices.kt
package com.mercadolibre.melikotlinmeetup.commons

import java.text.DecimalFormat
import java.text.NumberFormat
import java.util.*

fun Double.toPriceFormat() : String {
    val decimalFormat = NumberFormat.getNumberInstance(Locale.GERMAN) as DecimalFormat
    val formattedPrice = decimalFormat.format(this)

    return "$ $formattedPrice"
}